### PR TITLE
unpivoted table

### PR DESCRIPTION
**Pull request by AcckiyGerman**
script rewritten from scratch, readme and datapackage files updated.

### CODE REVIEW

File: scripts/process.py [hunk 1]
```diff
@@ -1,143 +1,82 @@
...
+                category = 'nominal_year'
+            elif 'Real' in value and 'Index' in unit:
+                category = 'real_index'
+            elif 'Real' in value and 'Year' in unit:
+                category = 'real_year'
+
+            output[category].writerow([parse_date(date), country[3:], price])
+
+
+def extract_csv_structure():
```
Comment (zelima) [inline scripts/process.py]: I'm not getting what this function is for. It seems is not even used

File: README.md [hunk 2]
```diff
@@ -17,40 +17,39 @@ Here we use *Selected series* set, reasons are:
...
+2012-06-30,Sweden,103.47
+```
 
 #### Detailed Data Description:
 
 Contains data for 59 countries at a quarterly frequency (real series are the nominal price series deflated by the consumer price index), both in levels and in growth rates (ie four series per country). These indicators have been selected from the detailed data set to facilitate access for users and enhance comparability. The BIS has made the selection based on the Handbook on Residential Property Prices and the experience and metadata of central banks. An analysis based on these selected indicators is also released on a quarterly basis, with a particular focus on longer-term developments in the May release.
 
 ## Preparation 
-You will need python3 installed to run the data downloading and processing script.
+You will need `git`, `python`, `pip` and `virtualenv` installed to run the data downloading and processing script.
```
Comment (zelima) [inline README.md]: * I don't think the virtual environment is necessary for this.
* One can download git repository as zip as well so no need to say `git` here
* Think simple python version would be enough here - no need of all that venv, git and pip
  * simply: you will need to run python3/2 installed on your machine to run script

**Comment by AcckiyGerman** [comment]
@zelima thanks for review.
Since we need to install tabulator so we need `pip`. And we don't need a virtual environment, if the task will work in a docker or similar thing. But it is recommended as a good habit to use venv, when you working locally. I've study it after several fails in other projects )))

**Comment by zelima** [comment]
@AcckiyGerman I understand it is good practice and I agree, but it's also making this complicated. I remind you that this is not only for developers who know coding well. Not everybody may be familiar with virtual environments and how to use them, especially beginners. The complication may scare many of them, so why make it that way? let's make it as simple as possible. 

Also, it says 
>You will need `git`, `python`, `pip` and `virtualenv` installed to run the data downloading and processing script."

While you don't actually "need", right? I mean it is good practice, but you don't have to have them to run the script right? Eg:
* you can download package as zip from Github - no need of `git`
* you can directly run `pip install ...` without venv and it will work - no need of `virtualenv`
* you may be able to install with conda or similar - no need of `pip`

**Pull request merged**